KomaMRIFiles/src/Phantom/Phantom.jl

```diff
@@ -167,7 +168,8 @@ end
 function export_motion!(motion_group::HDF5.Group, motion::ArbitraryMotion)
     HDF5.attributes(motion_group)["model"] = "ArbitraryMotion"
     HDF5.attributes(motion_group)["K"] = size(motion.dx)[2] + 1
-    motion_group["period_durations"] = motion.period_durations
+    motion_group["t_start"] = motion.t_start
+    motion_group["t_end"] = motion.t_end
 
     for key in ["dx", "dy", "dz"]
```

Comment:
Why do you do `t_start` and `t_end` differently?

Reply:
That is because at first, there we thought of three different ways for motion matrices to be stored: Explicit, Indexed or Default. To accommodate this, we needed an extra attribute with a string indicating the type of storing method. If you don´t like this, we can just omit it and simplify it, leaving the "Explicit" option only

Reply:
This was the schema created from what @pabloirarrazaval and you asked for:
![image](https://github.com/JuliaHealth/KomaMRI.jl/assets/93908925/d8542ddb-16c1-4155-a83f-5e633bebe02a)

If this is not desired now, I can change it to a more simple approach:
![image](https://github.com/JuliaHealth/KomaMRI.jl/assets/93908925/2c0a4074-9c20-482b-856d-41afdb3ed4d5)

Reply:
I didn't realize this was the reason. Indexed methods are fine, but I am unsure how useful they could be for the motion models, at the moment, both SimpleMotion and ArbitraryMotion apply to "all" spins (unless the arbitrary motion is incorrectly constructed, with less or more "spins").

Reply:
Solved in [e10474b](https://github.com/JuliaHealth/KomaMRI.jl/pull/408/commits/e10474b6f40c7118821a0da02b709a706d9e8509)